docs/lib/storage/fragments/js/copy.md

```diff
@@ -0,0 +1,136 @@
+## Copy
+
+`Storage.copy` method copies an existing file to a different key.
+
+It returns a `{key: new S3 Object key}` object on success:
+
+### Options
+
+```typescript
+await Storage.copy(
+	src: { 
+		key: string;
+		level?: 'private' | 'protected' | 'public';
+		identityId?: string; // only works if you are copying from a protected file 
+	},
+	dest: { 
+		key: string;
+		level?: 'private' | 'protected' | 'public' 
+	}, 
+	config?: {
+		track?: boolean, // if true, automatically send Storage Events to Amazon Pinpoint
+		progressCallback?: (progress) => any, // callback function on every successful part upload
+	}
+)
+```
+
+### Extra Options
+```typescript
+await Storage.copy({ key: 'src' }, { key: 'dest' }, {
+	acl?: string, // https://docs.aws.amazon.com/AWSJavaScriptSDK/latest/AWS/S3.html#upload-property
+	cacheControl?: string, // Specifies caching behavior along the request/reply chain
+	contentDisposition?: string, // Specifies presentational information for the object 
+	contentEncoding?: string, // Specifies what content encodings have been applied to the object and thus what decoding mechanisms must be applied  
+	contentLanguage?: string, // The language the content is in
+	contentType?: string, // A standard MIME type describing the format of the object data
```

Comment:
I would try to reuse it from the original object and added in case customer requires to change it